release-notes/opensearch-neural-search.release-notes-2.11.0.0.md

```diff
@@ -0,0 +1,9 @@
+## Version 2.11.0.0 Release Notes
+
+Compatible with OpenSearch 2.11.0
+
+### Features
```

Comment:
why multi model feature is not added in release notes?

Reply:
@martin-gaievski

Reply:
it went to directly to 2.x branch. I might need to update release note during backport.